projections/weather/spec/weather/endpoint_provider_spec.rb

```diff
@@ -7,5 +7,32 @@
 module Weather
   describe EndpointProvider do
     subject { EndpointProvider.new }
+
+    context "Endpoint set" do
+      let(:expected) do
+        {"endpoint"=>{"url"=>"https://example.com"}}
+      end
+
+      it 'produces the expected output from the EndpointProvider' do
+        params = EndpointParameters.new(**{:endpoint=>"https://example.com"})
+        endpoint = subject.resolve_endpoint(params)
+        expect(endpoint.uri).to eq(expected['endpoint']['url'])
+        expect(endpoint.headers).to eq(expected['endpoint']['headers'] || {})
+        expect(endpoint.properties).to eq(expected['endpoint']['properties'] || {})
```

Comment:
I guess we will populate `expected` with `headers` and `properties` in the future?

Reply:
These are based no the endpoint tests.  The tests for this ruleset dont have any headers or properties so nothing to test.

Reply:
Just realized that these are projections, haha. Not enough coffee. Please disregard.